products/BellHybrid/apps/common/include/common/models/BedtimeModel.hpp

```diff
@@ -65,7 +65,7 @@ namespace app::bell_bedtime
       public:
         BedtimeModel() = delete;
 
-        explicit BedtimeModel(ApplicationCommon *app, std::unique_ptr<AudioModel> &&audioModel)
+        explicit BedtimeModel(ApplicationCommon *app, std::shared_ptr<AbstractAudioModel> audioModel)
```

Comment:
Does `BedTimeModel` really need to have a shared pointer to `audioModel`? Shared pointers are much slower than unique pointers or references. Moreover, the cases where you need them are rather rare. In this case, a simple reference will be sufficient. This also applies to other models/presenters.